### DISCUSSION TITLE
I am facing an import issue when using swc in my project?

### DISCUSSION THREAD

**Discussion by siamahnaf** [swc-project/swc#7577] [Q&A] [answered]
my folder structure is-
![image](https://github.com/swc-project/swc/assets/78312010/61183bdc-6a66-44f9-9011-4b6d7eb728c1)

in src folder I have structure like-

`src > student > model > student.entity`
`src > user > model > user.entity`

I import user.entity into student.entity like -

`import { User } from "@/user/model/user.entity";`

I am getting an error like-

![image](https://github.com/swc-project/swc/assets/78312010/1dae2355-4ef2-41b9-b5c5-3b254ac9a350)

I create a `.swcrc` file like-

```
{
    "$schema": "https://json.schemastore.org/swcrc",
    "jsc": {
        "baseUrl": "./"
    }
}
```

and my `tsconfig.json`-

```
{
  "compilerOptions": {
    "module": "commonjs",
    "declaration": true,
    "removeComments": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "allowSyntheticDefaultImports": true,
    "target": "ES2021",
    "sourceMap": true,
    "outDir": "./dist",
    "baseUrl": "./",
    "paths": {
      "@/*": [
        "./src/*"
      ]
    },
    "incremental": true,
    "skipLibCheck": true,
    "strictNullChecks": false,
    "noImplicitAny": false,
    "strictBindCallApply": false,
    "forceConsistentCasingInFileNames": false,
    "noFallthroughCasesInSwitch": false,
  }
}
```

Where I am wrong?

**Comment by Gabrielcefetzada** [comment]
Did you solved it? Please, share with us the solution. I found this in NestJS docs
![image](https://github.com/swc-project/swc/assets/63877012/85da1b4e-a601-4ea4-8c9d-e69b3f7fee86)

But still getting the same error when i applied this possibility solution to all of my typeorm relations

**Comment by kdy1** [comment]
@siamahnaf @Gabrielcefetzada Please file an issue with a minimal reproduction.

**Comment by kdy1** [comment, marked as answer]
Oh you don't have `jsc.paths` in your `.swcrc`

**Reply by Gabrielcefetzada** [reply]
What should I put in my .swcrc file for it to work? I'm not understanding the problem.

**Comment by siamahnaf** [comment]
This is problem is only occured in windows.

**Reply by obe-ws** [reply]
@siamahnaf Do you use any ORM ? I just fixed the same issue by adding `Relation<ENTITY_NAME>` everywhere

**Comment by DuyetVu2001** [comment]
try to add this to tsconfig.json - it works with me
` "paths": {
      "@routes/*": [
        "./src/routes/*"
      ],
      "@middlewares/*": [
        "./src/middlewares/*"
      ]
    }`
